### PR TITLE
fix: cleanup wiki logic

### PR DESCRIPTION
**Pull request by raisedadead**
This commit:

- Removes the `help`, `menu`, `hello`, `welcome` commands
- Updates the `wiki` command for migration
- Updates the `rooms` command to point to forum article

Refers: #103

### CODE REVIEW

**Comment by raisedadead** [comment]
@BerkeleyTrue @QuincyLarson please QA

File: lib/bot/BotCommands.js [hunk 1]
```diff
@@ -84,31 +87,12 @@ const BotCommands = {
...
-    return 'type help for a list of things the bot can do';
-  },
-
   // TODO - sort alphabetically
   rooms: function() {
-    return '## rooms\nSee all the FreeCodeCamp rooms at ' +
-      '[gitter.im/FreeCodeCamp/rooms](https://gitter.im/orgs/FreeCodeCamp/' +
-      'rooms)\nOr check [this wiki article](https://github.com/freecodecamp/' +
-      'freecodecamp/wiki/official-free-code-camp-chat-rooms) for a shortlist';
+    return '#### Our chat rooms : ' +
```
Comment (dhcodes) [inline lib/bot/BotCommands.js]: I would change this to `freeCodeCamp rooms:`

File: lib/bot/BotCommands.js [hunk 2]
```diff
@@ -168,31 +152,20 @@ const BotCommands = {
...
-      return '## welcome to FreeCodeCamp @' +
-        input.message.model.fromUser.username + '!';
-    }
-    return null;
-  },
-
-  hello: function(input, bot) {
-    return (this.welcome(input, bot) );
+  wiki: function() {
+    return '#### Our wiki : ' +
```
Comment (dhcodes) [inline lib/bot/BotCommands.js]: I recommend changing this to `freeCodeCamp Wiki:`

File: lib/bot/BotCommands.js [hunk 3]
```diff
@@ -168,31 +152,20 @@ const BotCommands = {
...
+    '\n:point_right: Our wiki has moved to ' +
```
Comment (dhcodes) [inline lib/bot/BotCommands.js]: How long should we acknowledge that the wiki has a new home? Perhaps change `Our wiki has move to` to `The freeCodeCamp wiki can be found on`

**Comment by QuincyLarson** [comment]
@dhcodes have you gotten this running and verified everything works OK? If so, could you merge this? Then we can ask @BerkeleyTrue to deploy @raisedadead's fixes.

**Comment by dhcodes** [comment]
I didn't test these latest changes because my bot setup is on another machine but assuming it's just string edits, I think this is good to go!

**Comment by QuincyLarson** [comment]
@raisedadead Awesome. Thank you for shutting off the greeting feature. Chat is much more fluid now!

**Pull request merged**